clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java

```diff
@@ -1025,10 +1025,10 @@ public void listSessions(Run run, boolean includeInactive, BiConsumer<AgentInfo,
          Handler<AsyncResult<Void>> completionHandler) {
       invokeOnAgents(run, AgentControlMessage.Command.LIST_SESSIONS, includeInactive, completionHandler, (agent, result) -> {
          if (result.failed()) {
-            log.error("Agent " + agent + " failed listing sessions", result.cause());
+            log.error("Agent {} failed listing sessions", agent, result.cause());
             sessionStateHandler.accept(agent, "");
          } else if (result.result() instanceof Throwable) {
-            log.error("Agent " + agent + " has thrown an error while listing sessions", (Throwable) result.result());
+            log.error("Agent {}} has thrown an error while listing sessions", agent, (Throwable) result.result());
```

Comment:
```suggestion
            log.error("Agent {} has thrown an error while listing sessions", agent, (Throwable) result.result());
```

Reply:
Thank you for answer
I will quickly fix it and PR it for you!